setup.py

```diff
@@ -61,6 +61,8 @@ def get_version():
             "qblox-instruments==0.7.0",
             "qcodes",
             "pyvisa-py==0.5.3",
+            "qm-qua",
+            "qualang-tools",
```

Comment:
In view of a poetry transformation, let me suggest to have a minimum required version here.

Reply:
Done, I used the latest versions.